lib/commands/deploynodeapp.js

```diff
@@ -72,6 +80,31 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'r',
+    required: false
+  },
+  'node-version': {
```

Comment:
Oh yeah, that's right.  This is a _node.js-specific command_.  I guess I can live with it for now.